phpcr-shell/querying.rst

```diff
@@ -169,6 +169,40 @@ Arguments:
   property
 - **value**: Value to append
 
+expr
+""""
+
+Evaluate an expression. This function is very powerful in that it enables
+you to use the `Symfony Expression Language`_ to evaluate an expression, the
+result of which can be assigned to a property.
+
+Within the expression you have access to the `row` object
+(`RowInterface`_).
+
+Set the value of ``a.title`` to the node name:
+
+.. code-block:: bash
+
+    PHPCRSH> UPDATE [nt:unstructured] AS a SET a.title = expr('row.getNode().getName()')
+
+Set the value of ``a.title`` to the value of the property :
+
+.. code-block:: bash
+
+    PHPCRSH> UPDATE [nt:unstructured] AS a SET a.title = expr('row.getNode().getName()')
```

Comment:
this is the same as above. did you want to say something like row.getNode().getPropertyValue("subject")